### DISCUSSION TITLE
MP600 TLC 100gb cache and seagate firecuda 520 TLC 100gb cache plotting idea?

### DISCUSSION THREAD

**Discussion by PC101guy** [Chia-Network/chia-blockchain#5239] [Ideas]
It is a shock and surprise to many people that news is going around that these drives that were recommended by the chia dev team are now known to the community as subpar drives with some of the slowest sustained write speeds of any nvme's on the market!  I ordered 4 2tb seagate firecuda 520 costing close to $2000.00 Canadian because They were recommended by chia and now I am forced to refund these drives as they are just not meant for chia plotting and even being gen 4.0 under perform even against gen 3.0 and drop to a measly 500mb/s write speed once the built in 100gb cache is filled.    It would seem I am not alone, there are not many sites that currently have these drives in especially the 2tb variant so pretty soon as people realize there will be more and more rage and disappointment in these drives being suggested as an ideal chia plotting drive.  Now I am not an expert at all, not even close, i don't understand how nvme's work and how they work with chia plotting, but is there not a way to code into the plotting process that would allow these drives to fill up their cache at full speed, and then give them time to clear the cache  and resume plotting again and repeat over and over until the plot is done? is that beyond the limits of the plotting system? Is it just impossible and what im asking just doesn't make sense in a technical standpoint.  I was doing a lot of research and the cache is suppose to empty itself when writing stops but im not sure how long it takes to empty before the speeds go back full capacity of these drives.    Basically is there no way to use these slc drives to their fullest capability for chia plotting? it seems like such a waste for people who are over paying for these drives because of chia's recommendations.  or are they doomed to become the laughing drives of the chia plotting world?  Thanks.

Source for stated speed drop after cache filled:
https://www.youtube.com/watch?v=lv2avfUiBKk&t=304s  @4:10 mark for graph.

**Comment by PC101guy** [comment]
The plotter should have the ability to fill the cache and allow it to empty before refilling it again.  it may take a few seconds to empty the cache (literally guessing) but The benefits would most likely be huge, going from 4,300 mb/s write to 500mb/s is a huge drop in performance especially when it drops off rather quickly and then stays at the very limited 500mb/s for the remainder of the plot, and if the job had to be started and stopped a few dozen times per plot to allow the cache to empty it would still be faster running at the full 4,300 mb/s would it not? let me know your thoughts and if this idea has merit or is impossible.  It all comes down to not "overflow the cache" the built in cache,  also if you used these drives in raid 0 would the cache also be multiplied for instance, running 4x 1tb mp600 each with 100gb cache total=400gb would require less idle time for cache to empty then running 4 separate drives with 100gb each.  Ok now im just brain storming again not an expert!

**Comment by k3vst3r** [comment]
Believe recommended due to endurance rating, ssd manufacturers will eventually make chia specific plotting drives designed to write plotting efficiently.
